medicines/doc-index-updater/src/create_manager/sanitiser.rs

```diff
@@ -0,0 +1,114 @@
+#[derive(Clone)]
+pub struct SanitisedString {
+    inner: String,
+}
+
+impl From<String> for SanitisedString {
+    fn from(inner: String) -> Self {
+        Self { inner }
+    }
+}
+
+impl From<&String> for SanitisedString {
+    fn from(inner: &String) -> Self {
+        Self::from(inner.to_owned())
+    }
+}
+
+impl From<&str> for SanitisedString {
+    fn from(inner: &str) -> Self {
+        Self::from(inner.to_owned())
+    }
+}
+
+impl ToString for SanitisedString {
+    fn to_string(&self) -> String {
+        self.inner
+            .replace(|c: char| !c.is_ascii(), "")
+            .replace("\n", " ")
+            .trim()
+            .to_string()
+    }
+}
+
+#[derive(Clone)]
+pub struct VecSanitisedString {
+    inner: Vec<String>,
+}
+
+impl From<Vec<String>> for VecSanitisedString {
+    fn from(inner: Vec<String>) -> Self {
+        Self { inner }
+    }
+}
+
+// impl Iterator for VecSanitisedString {
+//     type Item = SanitisedString;
+//     fn next(&mut self) -> Option<Self::Item> {
+//         if let Some(a) = self.iter {
+//             //
+//         } else {
+//             self.iter = self.inner.iter();
+//         }
+
+//         if let Some(a) = self.iter.next() {
+//             Some(SanitisedString::from(a))
+//         } else {
+//             self.iter = None;
+//             None
+//         }
+//     }
+// }
```

Comment:
Why is this here?

If there's a reason for it, there should be a comment to explain.

Reply:
nope, not needed, will kill